Sourcery/Sourcery.swift

```diff
@@ -96,7 +96,7 @@ class Sourcery {
 
         Log.info("Starting watching sources.")
```

Comment:
Maybe we should add here list of directories that are being observed?

Reply:
why ? this doesn't change the behaviour of the watcher, all the files you were observing previously would still be observed, this only changes the implementation detail, since any file changes will cause regeneration like they did before

Reply:
Just for better visibility what is happening. Also when new folder is added and it is started being watched we may log this event too, for the same reason. Maybe in a more verbose mode of watcher.
To make sure I don't misunderstood the change. Watcher now will watch only folders and not folders and individual files in it? But events will still come in when files change, right?

Reply:
Some time ago we broke the following behaviour, that I wanted to work (and it did work in the past):

- When a new code or template is added, it should be immediately supported in watcher mode so you don't need to restart Sourcery session

Watcher will observe all top level directories, observing directory still generates file events. It shouldn't change anything for the end-user except again supporting the above scenario

Reply:
Ok, got it =) Would still add folder names to the log, but that's up to you =)